### PR TITLE
Alyx panel test

### PR DESCRIPTION
**Pull request by k1o0**
- Added test class for AlyxPanel
- Last weight of the day now used for calculating percent weight
- Bug fix for running timer in stand-alone instance
- Login can now occur without input dialog
- Bug fix for error when only two weights exist, both on the same day

### CODE REVIEW

**Comment by jaib1** [comment]
why did you delete all those comments from the todo bot, and turn this pull request into a draft?

**Comment by k1o0** [comment]
The only comments I deleted were ones that I myself have resolved in the code.  I don't remember what the comments were if there were some that I didn't delete but have vanished :/

I think I opened this pull request as a draft.  I don't think it's ever been open for review... I'll open it when enough of the tests are written.

**Comment by jaib1** [comment]
oh I see. I was thinking github had a "resolve" feature for comments (like Google Docs), as opposed to just deleting comments, but I guess not?

**Comment by k1o0** [comment]
It has resolves for in-line code comments but not for the conversation tab

**Comment by k1o0** [comment]
This is now ready for review and all tests pass on my end.  You may want to merge cortex-lab/alyx-matlab#42 first

File: tests/AlyxPanel_test.m [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+      delete(testCase.Panel)
+      % Double check no figures left
+      figHandles = findobj('Type', 'figure');
+      if ~isempty(figHandles)
+        idx = cellfun(@(n)any(strcmp(n, testCase.Subjects)),{figHandles.Name});
+        close(figHandles(idx))
+      end
+      % Remove subject directories
+      dataRepo = getOr(dat.paths, 'mainRepository');
+      assert(rmdir(dataRepo, 's'), 'Failed to remove test data directory')
```
Comment (jaib1) [inline tests/AlyxPanel_test.m]: this line fails.

it seems no 'Subjects' folder is found?

**Pull request merged**